src/components/appointments/CreateAppointment/AppointmentProgress.js

```diff
@@ -47,26 +50,59 @@ const AppointmentProgress = ({title, currentStep}) => {
     };
   });
 
-  const ProgressIndicatorDisplayComponent =
-    config?.displayComponents?.progressIndicator ?? ProgressIndicatorDisplay;
+  // Add the fixed steps to the the original steps array
+  const stepsToRender = addFixedSteps(
+    steps,
+    submission,
+    currentPathname,
+    showOverview,
+    showConfirmation,
+    isSubmissionComplete
+  );
+
+  // Figure out the title for the mobile menu based on the state
+  let activeStepTitle;
+  if (isStartPage) {
+    activeStepTitle = intl.formatMessage({
+      description: 'Start page title',
+      defaultMessage: 'Start page',
+    });
+  } else if (isSummary) {
+    activeStepTitle = intl.formatMessage({
+      description: 'Summary page title',
+      defaultMessage: 'Summary',
+    });
+  } else if (isConfirmation) {
+    activeStepTitle = intl.formatMessage({
+      description: 'Confirmation page title',
+      defaultMessage: 'Confirmation',
+    });
```

Comment:
the translations/literals can be defined in a shared location using `react-intl`'s `defineMessage` :)